frontend/components/auction/surplus/SurplusAuctionTransactionFlow.stories.js

```diff
@@ -102,9 +100,10 @@ storiesOf('Auction/Surplus/SurplusAuctionTransactionFlow', module)
             connect() {
                 this.isConnectingWallet = true;
                 setTimeout(() => {
-                    this.walletMKR = new BigNumber(faker.finance.amount(1, 2));
+                    this.walletMKR = new BigNumber(0);
```

Comment:
Why are you setting this to 0? If we want to test the flow, the wallet has to have some MKR to bid.